proofs/signatures/th_lira.plf

```diff
@@ -52,3 +52,21 @@
     ((/_Real x y) (/_Real (reify_real_term x) (reify_real_term y)))
   ))
 
+; Predicates of the form (term Integer) >= (term Real) or negations thereof
+(declare >=_IntReal (! x (term Int) (! y (term Real) formula)))
+
+; From an arith predicate, compute the equivalent real predicate
+; All arith predicates are (possibly negated) >='s with a real on the right.
+; Technically it's a real literal on the right, but we don't assume that here.
+(program reify_arith_pred ((p formula)) formula
+  (match p
+         ((not p') (match p'
```

Comment:
Maybe I am too naive here but can't we do
```
((not p') (not (reify_arith_pred p'))
```
?

Reply:
Well yes. That is a good idea :P.